### DISCUSSION TITLE
Can I add open source to my protfolio?

### DISCUSSION THREAD

**Discussion by MaorBemdoo** [EddieHubCommunity/support#8119] [Q&A] [answered]
So on my portfolio website can I say that I'm an open source contributor and I've been doing it since `xxxx` especially on my experiece section

**Comment by Vijaykv5** [comment, marked as answer]
Yes, you can absolutely include your  **open-source contributions** in your portfolio website's experience section. Open-source contributions demonstrate your skills, dedication, and collaboration abilities, making them valuable additions to your professional profile.

It will help the recruiters or the employers of the particular company hiring to understand your expertise and commitment towards the open-source community.
Also adding your open-source contributions to your portfolio is a great way to showcase your skills and demonstrate your active participation in the tech community.

**Reply by MaorBemdoo** [reply]
Tnks alot, this was really helpful

**Comment by martcpp** [comment]
Adding Open source to your portfolio shows the recruiter how sn build in team and your team strength especially is the prince is a large one

**Comment by CBID2** [comment]
> So on my portfolio website can I say that I'm an open source contributor and I've been doing it since `xxxx` especially on my experiece section

I second @Vijaykv5's comment. Open source is a great tool to add to your portfolio. I highly recommend checking out [OpenSauced](https://opensauced.pizza/about). They have a feature called Highlights, where you can display your merged PRs, issues, and blog posts. You can learn more about it here: https://docs.opensauced.pizza/community/highlights/

**Comment by sarthakkarora** [comment]
Feel free to add it to your portfolio if you've made significant changes or raised issues that lead to enhancements. However, avoid including unresolved or minor issues, as they reflect more on someone else's project than your own contributions. I don't mean to discourage you... contributing to open source is great! Adding such work to your profile shows how skilled you are at finding bugs or improving code.
